imports/ui/ranking/Ranking.js

```diff
@@ -26,5 +130,37 @@ Template.Ranking.events({
 
         Session.set("selectedStatistics", statistics);
         $("#modal").modal('show');
+    },
+
+    /**
+     * Event handler for modifying ranking sort statistic
+     * @param[in] event The event to handle
+     */
+    'change select[name="sortStatistic"]'(event) {
```

Comment:
I have no idea what's going on here lmao

Reply:
Apparently this is the syntax
'change -> Detect a "change event"
select -> on an HTML select field
[name="sortStatistic"]' -> with the name "sortStatistic"
(event) -> capture the event

Reply:
I'm like, super confused how this is calling a function with a string name? Or I guess this is a function definition. JS is weird

Reply:
JS is the worst